errors.go

```diff
@@ -70,13 +70,27 @@ func newCellNameToCoordinatesError(cell string, err error) error {
 	return fmt.Errorf("cannot convert cell %q to coordinates: %v", cell, err)
 }
 
+// newColumnNumberError defined the error message on receiving invalid column
+// number.
+func newColumnNumberError(count int) error {
```

Comment:
Use the `col` instead of `count` for better semantic, because this function is only used internally, it can be simplified like this:

```go
func newColumnNumberError(col int) error {
    if col < MinColumns {
        return fmt.Errorf("column number less than the minimum limit %d", MinColumns)
    }
    return fmt.Errorf("column number exceeds the maximum limit %d", MaxColumns)
}
```